.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+name: OpenWISP Config CI Build
+
+on:
+  push:
+    branches:
+      -master
+  pull_request:
+    branches:
+      -master
+  
+jobs:
+
+  build:
+    name: QA-Checks and Tests
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```
Building on Ubuntu 20.04 should not raise any problems.